### DISCUSSION TITLE
Publish express api app to vercel

### DISCUSSION THREAD

**Discussion by WebsiteHostJarno** [vercel/community#273] [Help] [answered]
My code structure you can find here when i deploy the building loads infinity

https://github.com/WebsiteHostJarno/apis

@everyone

**Comment by gogocharli** [comment, marked as answer]
Hello @WebsiteHostJarno, in order to publish your Express App to Vercel, you would need to move the contents of that repository inside an `/api` folder along with a `vercel.json` file to redirect content properly to your app.

More details can be found in [this article](https://dev.to/hte305/deploy-express-js-app-to-vercel-38jb). You can use the linked repository[^1] as a guide.

[^1]: https://github.com/hatuanem199801/express-vercel

**Reply by WebsiteHostJarno** [reply]
@gogocharli i gion to try it
